rocAL/rocAL/source/graph.cpp

```diff
@@ -28,17 +28,17 @@ THE SOFTWARE.
 
```

Comment:
As we discussed please move all pipeline/graph files into pipeline folder

Reply:
Resolved